### ISSUE TITLE
Science Sleuth error response

### ISSUE THREAD

**Issue by aaronschachter** [DoSomething/ds-mdata-responder#520]
Currently texting in REWARD results in a "Oof! something went wrong" because the opt-in paths are looking for custom fields which no longer exist. 

This isn't an issue we necessarily need to make any code changes for, this can be closed out once we add the Custom Fields we need into Mobile Commons and verify things work.

From [`DonationController.js`](https://github.com/DoSomething/ds-mdata-responder/blob/2a837e05bb995c098d1c3097ff9ec55e5ff9bbda/app/lib/donations/controllers/DonorsChooseDonationController.js#L171):
````
        var mobileCommonsCustomFields = {
          SS_fulfillment_trailer:   entities.decode(selectedProposal.fulfillmentTrailer), 
          SS_teacher_name :         entities.decode(selectedProposal.teacherName), 
          SS_school_name :          entities.decode(selectedProposal.schoolName),
          SS_proj_location:         location
        };
````

cc @deadlybutter @jamjensen @classicfreddie 

<img width="655" alt="custom_fields" src="https://cloud.githubusercontent.com/assets/1236811/16602786/b222759c-42cb-11e6-85d0-b0044d5dbcde.png">

**Comment by aaronschachter** [comment]
Mobile Commons has added all missing custom fields, but still get an "Oof" message back.  Here's what showed up in the Heroku logs, will start messing with this endpoint locally.

> 2016-07-13T22:27:25.379961+00:00 heroku[router]: at=info method=POST path="/donations/donors-choose/start?id=201" host=ds-mdata-responder.herokuapp.com request_id=7518884e-8c32-4660-81d8-e0e80a6cc5e1 fwd="64.22.127.230" dyno=web.1 connect=1ms service=13ms status=200 bytes=131

**Comment by aaronschachter** [comment]
Inspecting error logs show this for my number when I text in REWARD:
````
016-08-01T22:04:39.344963+00:00 app[web.2]: [31merror[39m: Invalid JSON data received from DonorsChoose API for user mobile: [my number] , or selected proposal does not contain necessary fields. 
Error: Error: No proposals returned from Donors Choose
````

**Comment by aaronschachter** [comment]
Didnt realize the "Oof message" comes from the conversation copy set on the Donation Flow MoCo campaign's [error_start_again Optin Path] (https://secure.mcommons.com/campaigns/137049/opt_in_paths/191511). 

Adding the custom fields into Mobile Commons didn't fix because the DonorsChoose API query wasn't returning any results (now fixed by #545).
